app/api/auth.py

```diff
@@ -23,6 +24,11 @@
 from app.models.user import User
 
 auth_routes = Blueprint('auth', __name__, url_prefix='/v1/auth')
+with app.app_context():
+    db.init_app(app)
+    db.create_all()
+    twitter_blueprint = make_twitter_blueprint(api_key=TwitterOAuth.get_client_id(),
```

Comment:
Remove this

Reply:
Will remove it. But Travis fails upon doing that. How do we make it pass?

Reply:
Initialize this in create_app

Reply:
Or see any sample app of Flask Dance